Resources/config/services.yml

```diff
@@ -17,3 +23,21 @@ services:
             - "@?logger"
         tags:
             - {name: doctrine.event_subscriber}
+    algolia.metadata.file_locator:
+        public: false
+        class: "%algolia.metadata.file_locator.class%"
+        arguments:
+            - []
+    algolia.metadata.annotation_loader:
+        class: "%algolia.metadata.annotation_loader.class%"
+    algolia.metadata.chain_loader:
+        public: false
+        class: "%algolia.metadata.chain_loader.class%"
+        arguments:
+            - [ "@algolia.metadata.yaml_loader", "@algolia.metadata.annotation_loader" ]
```

Comment:
I'm not sure hardwiring this is a good idea - maybe expose this in the configuration?